.github/workflows/pipeline-deploy-account-infrastructure.yaml

```diff
@@ -20,7 +20,7 @@ jobs:
           environment: mgmt
           domain: "uec-dos-management"
           workspace: ${{ needs.metadata.outputs.workspace }}
-          stacks: "['terraform_management','github-runner','dos-artefacts']"
+          stacks: "['terraform_management','github-runner','dos-artefacts','parameter_store']"
```

Comment:
Interesting... I think there is a dependency between dos-artefacts and parameter_store. The parameters need to be created AND the default values updated with actual account ids BEFORE dos-artefacts. I think first time thru the dos-artefacts will get built with the  placeholder values inserted by terraform into system parameters. We would need to update the parameters with the actual account ids and rerun the workflow to update the dos-artefacts not the account ids.  We would have the same problem if we had a separate earlier job to just do the parameter store stack... Unless anyone can think of a smart way around this ? @gdcNHS @jope1 @tamaragoldschmidt

Reply:
is this a similar issue to the artefact buckets - you need to deploy in a 2 step process?

Reply:
yes I think so but not sure I can see a way around it - more of a DR issue perhaps. Rather than holding us up I think we will go for now at least with the do it twice approach